codespell_lib/data/dictionary.txt

```diff
@@ -17752,6 +17752,8 @@ navagate->navigate
 navagating->navigating
 navagation->navigation
 navagitation->navigation
+nax->max
+naximum->maximum
```

Comment:
I guess there's also the 'maxima' and 'maximal' versions, if you're going for completeness.
https://en.wikipedia.org/wiki/Maxima_and_minima

Reply:
Done.